### DISCUSSION TITLE
LDT tile dimension not matching

### DISCUSSION THREAD

**Discussion by Arijit22IITDelhi** [NASA-LIS/LISF#1528] [Q&A] [answered]
Hello, I was trying to run LDT for ensemble generation (from 1 to 20). The input restart file that I am using contains "ntiles: 71720". But after running the LDT config file with almost all the necessary specifications, it is showing an error as per the LDT log entry below:

- - - - - - - - - Noah-MP Parameters - - - - - - - - - - - -
 Total number of surface tiles: 71623
 Total number of tiles for LSM: 71623
 Total number of grid points:       71623
  Generating restart file: ./OUTPUT/LIS_EnRST_JOINT_DA_Trial.nc
 [ERR] The tile dimension in the input restart file
   does not match the tile dimension computed in LDT.
 endrun is being called

How do match the number of tiles for both? I am sharing the corresponding LDT config file and the error log file for reference.

[ldt_enrst.txt](https://github.com/NASA-LIS/LISF/files/15126908/ldt_enrst.txt)
[ldtlog_Enrst.txt](https://github.com/NASA-LIS/LISF/files/15126911/ldtlog_Enrst.txt)

**Comment by MariRiba** [comment]
Hi
I also encountered similar error before. And I solved it by matching the config file for ensemble-restart and config file for input restart . Can you provide config file that generated the  input restart file ?

**Reply by Arijit22IITDelhi** [reply]
@MariRiba sure, here is the lis.config file, from which I generated the corresponding input restart file for LDT-ensemble restart:

[LIS_MERRA2.txt](https://github.com/NASA-LIS/LISF/files/15129992/LIS_MERRA2.txt)

**Reply by MariRiba** [reply, marked as answer]
Sorry, @Arijit22IITDelhi 
The lis_input***.nc file is generated by the ldt.config . You might have mismatch in that file and ensemble restart config.
Can you provide that file too?

**Reply by Arijit22IITDelhi** [reply]
Ok, now I have the reason, the issue got resolved. Thank you, @MariRiba.

**Reply by Shuzunyun** [reply]
> Ok, now I have the reason, the issue got resolved. Thank you, @MariRiba.

Hi, how did you solve this problem?  I encountered the same problem as you did.
